dll/win32/shell32/wine/PolicyData.h

```diff
@@ -0,0 +1,307 @@
+DEFINE_POLICY( REST_NOMOVINGBAND,                   "Explorer",         "NoMovingBands"                    ),
+#endif
+
+DEFINE_POLICY( REST_NOEDITDESKCOMP,                 "ActiveDesktop",    "NoEditingComponents"              ),
+DEFINE_POLICY( REST_NORESOLVESEARCH,                "Explorer",         "NoResolveSearch"                  ),
+DEFINE_POLICY( REST_NORESOLVETRACK,                 "Explorer",         "NoResolveTrack"                   ),
+DEFINE_POLICY( REST_FORCECOPYACLWITHFILE,           "Explorer",         "ForceCopyACLWithFile"             ),
+
+#if (WINE_FILEVERSION_MAJOR <= 6) && (NTDDI_VERSION < NTDDI_LONGHORN)
+DEFINE_POLICY( REST_NOLOGO3CHANNELNOTIFY,           "Explorer",         "NoMSAppLogo5ChannelNotify"        ),
+#endif
+
+DEFINE_POLICY( REST_NOFORGETSOFTWAREUPDATE,         "Explorer",         "NoForgetSoftwareUpdate"           ),
+DEFINE_POLICY( REST_NOSETACTIVEDESKTOP,             "Explorer",         "NoSetActiveDesktop"               ),
+DEFINE_POLICY( REST_NOUPDATEWINDOWS,                "Explorer",         "NoWindowsUpdate"                  ),
+DEFINE_POLICY( REST_NOCHANGESTARMENU,               "Explorer",         "NoChangeStartMenu"                ),
+DEFINE_POLICY( REST_NOFOLDEROPTIONS,                "Explorer",         "NoFolderOptions"                  ),
+DEFINE_POLICY( REST_HASFINDCOMPUTERS,               "Explorer",         "FindComputers"                    ),
+DEFINE_POLICY( REST_INTELLIMENUS,                   "Explorer",         "IntelliMenus"                     ),
+DEFINE_POLICY( REST_RUNDLGMEMCHECKBOX,              "Explorer",         "MemCheckBoxInRunDlg"              ),
+DEFINE_POLICY( REST_ARP_ShowPostSetup,              "Uninstall",        "ShowPostSetup"                    ),
+DEFINE_POLICY( REST_NOCSC,                          "Explorer",         "NoSyncAll"                        ),
+DEFINE_POLICY( REST_NOCONTROLPANEL,                 "Explorer",         "NoControlPanel"                   ),
+DEFINE_POLICY( REST_ENUMWORKGROUP,                  "Explorer",         "EnumWorkgroup"                    ),
+DEFINE_POLICY( REST_ARP_NOARP,                      "Uninstall",        "NoAddRemovePrograms"              ),
+DEFINE_POLICY( REST_ARP_NOREMOVEPAGE,               "Uninstall",        "NoRemovePage"                     ),
+DEFINE_POLICY( REST_ARP_NOADDPAGE,                  "Uninstall",        "NoAddPage"                        ),
+DEFINE_POLICY( REST_ARP_NOWINSETUPPAGE,             "Uninstall",        "NoWindowsSetupPage"               ),
+DEFINE_POLICY( REST_GREYMSIADS,                     "Explorer",         "GreyMSIAds"                       ),
+DEFINE_POLICY( REST_NOCHANGEMAPPEDDRIVELABEL,       "Explorer",         "NoChangeMappedDriveLabel"         ),
+DEFINE_POLICY( REST_NOCHANGEMAPPEDDRIVECOMMENT,     "Explorer",         "NoChangeMappedDriveComment"       ),
+DEFINE_POLICY( REST_MaxRecentDocs,                  "Explorer",         "MaxRecentDocs"                    ),
```

Comment:
Is it intended that REST_MaxRecentDocs and REST_ARP_ShowPostSetup are not all capitalized like all the other ones?

Reply:
Yes. It is intended.
Same as `<shlobj.h>`'s ones.

Reply:
Check the official headers @JoachimHenze